Cargo.toml

```diff
@@ -28,7 +28,7 @@ regex = "1"
 structopt = "0.3"
 crates-index = { version = "0.18.5", optional = true }
 rayon = { version = "1", optional = true }
-crates-index-diff = "10.0.0"
+crates-index-diff = "12.0.0"
```

Comment:
this should be `12.1.0`, right?

Reply:
Good catch! Fixed in c2cd2b7b70ab0e72a01c90fe7cec01e5d9ed53da.